parse/fake_data_parse.go

```diff
@@ -89,22 +94,22 @@ func (fdp FakeDataParse) ReplaceVars(req *definition.Request, input string) stri
 func (fdp FakeDataParse) Parse(req *definition.Request, res *definition.Response) {
 	for header, values := range res.Headers {
 		for i, value := range values {
-			res.Headers[header][i] = fdp.ReplaceVars(req, value)
+			res.Headers[header][i] = fdp.ReplaceVars(req, res, value)
 		}
 
 	}
 	for cookie, value := range res.Cookies {
-		res.Cookies[cookie] = fdp.ReplaceVars(req, value)
+		res.Cookies[cookie] = fdp.ReplaceVars(req, res, value)
 	}
 
-	res.Body = fdp.ParseBody(req, res.Body, res.BodyAppend)
+	res.Body = fdp.ParseBody(req, res, res.Body, res.BodyAppend)
```

Comment:
If we already send the res param, maybe we can use only two params instead of four. What do you think?